### PR TITLE
Task1.1 solution

### PR DESCRIPTION
**Pull request by Vomchick**

### CODE REVIEW

File: Task1.1/Program.cs [hunk 1]
```diff
@@ -0,0 +1,25 @@
+﻿namespace Task1._1
+{
+    internal class Program
+    {
+        static void Main(string[] args)
```
Comment (IgorKheidorov) [inline Task1.1/Program.cs:5]: b can be less then a, order is not defined in a task. Values can be negative

**Review by IgorKheidorov — CHANGES_REQUESTED** [review]
Good solution, just fix issue

File: Task1.1/Program.cs [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+            int counter = 0;
+            for (int i = smaller; i <= bigger; i++)
+            {
+                int j = i;
+                while (Math.Abs(j) > 11)
+                {
+                    if (Math.Abs(j % 12) == 10) 
+                        counter++;
+                    j /= 12;
+                }
```
Comment (Vomchick) [inline Task1.1/Program.cs:24]: Can if statement be used without {}, just with one enter? Because using brackets will use 2 more rows 😊

Comment (Vomchick) [reply Task1.1/Program.cs:24]: Heh, you answered the question on the lecture, even when you didn't know about it. Thanks

**Review by IgorKheidorov — APPROVED** [review]

**Pull request merged**